src/main/java/sc/fiji/bdvpg/sourceandconverter/display/BrightnessAdjuster.java

```diff
@@ -0,0 +1,49 @@
+package sc.fiji.bdvpg.sourceandconverter.display;
+
+import bdv.util.BdvHandle;
+import bdv.viewer.SourceAndConverter;
+import net.imglib2.converter.Converter;
+import net.imglib2.display.ColorConverter;
+
+import java.util.function.Consumer;
+
+public class BrightnessAdjuster implements Runnable, Consumer< SourceAndConverter > {
+
+    BdvHandle bdvHandle;
+    SourceAndConverter< ? > sac;
+    double min;
+    double max;
+
+    public BrightnessAdjuster( final BdvHandle bdvHandle, final SourceAndConverter< ? > sac, double min, double max )
```

Comment:
Because the convertersetup is not passed to bdv, one has to call this action to change brightness and contrast. I'm ok with that, but then it means we need to provide a new convenient ui.

Also one remark : if a source and converter is displayed in multiple bdvhandle, one need to call this action as many times as there are bdv windows displaying this sourceandconverter object.